doc/releases.rst

```diff
@@ -23,15 +23,20 @@ Timeline
 |                |Development|`Dumpling`_|`Emperor`_ |`Firefly`_ |`Giant`_   |`Hammer`_  |`Infernalis`_ |`Jewel`_   |`Kraken`   |
 |                |Testing    |LTS        |Stable     |LTS        |Stable     |LTS        |Stable        |LTS        |Stable     |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
-| January   2017 |           |           |           |           |           |`0.94.10`_ |              |           |`11.2.0`_  |
+| February  2017 | `12.0.0`_ |           |           |           |           |`0.94.10`_ |              |           |           |
++----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
+| January   2017 |           |           |           |           |           |           |              |           |`11.2.0`_  |
 +----------------+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
 | December  2016 |           |           |           |           |           |           |              |`10.2.4`_  |           |
++                +-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+
+|                |           |           |           |           |           |           |              |`10.2.5`_  |           |
```

Comment:
The `+-----------+-----------+-----------+-----------+-----------+-----------+--------------+-----------+-----------+` line doesn't look long enough?